### PR TITLE
/desktop refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied refresh as per [doc](https://docs.google.com/document/d/1ZhMkLlkXcimHTIe1taXy24vbbR9pBkZd49H2sJqeQH0/edit#heading=h.fqpazxhydeus) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?node-id=613%3A501&mode=dev)
- Applied /desktop/features -> /desktop redirect as per ticket
- Bumped Vanilla version

## QA

- View the site locally in your web browser at: https://ubuntu-com-13757.demos.haus/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9643

### CODE REVIEW

**Comment by juanruitina** [comment]
A few comments, most visual related 🙃

- [ ] Please make the Windows logo same size as the other two.
- [ ] The hero seems to have excessive padding below, more than it used to, can you tweak it?
- [ ] Can we avoid wrapping of Certified logos in smaller screens?

Screenshot 2024-04-17 at 16 47 43

- [ ] There is a double HR on smaller devices above Ask Ubuntu

Screenshot 2024-04-17 at 16 41 42

- [ ] Please turn the H5-looking headings under "Enterprise-grade support and integration" and "Backed by a global community" into actual H3s, as is the case in the other sections.
- [ ] Please lowercase "today" in "Get started with Ubuntu today"
- [ ] The sections under "Get started with Ubuntu today" have weird spacing. I'd think paragraphs don't need to be wrapped in a `p-section--shallow` div?

I wonder if the "Thousands of applications" section should be deep, it feels too close to the next section, @lyubomir-popov wdyt?

**Comment by mtruj013** [comment]
Thanks @juanruitina, I addressed your comments please take another look. @lyubomir-popov could you also take a look please?

File: templates/shared/contextual_footers/_desktop.html [hunk 1]
```diff
@@ -1,40 +0,0 @@
-<section class="p-section">
```
Comment (mtruj013) [inline templates/shared/contextual_footers/_desktop.html:1]: Deleted the partial as there are no pages besides the /desktop index using it

**Comment by lyubomir-popov** [comment]
- please add`is-muted` to all but the first `<hr>`s in each section, so the hr above h2s stands out more than the rest. We'll need to think of the best way to document that on vailla, as it should apply to all pages.

- p-equal-heights already comes with a section padding-bottom, no need for the outer p-section here: 

image

- shallow section here please 
image

- no need for the last p-section--shallow class as there is one on the parent 
image

**Pull request merged**